test/src/tests/glide.yaml

```diff
@@ -1,7 +1,7 @@
 package: tests
 import:
 - package: github.com/Azure/go-autorest
-  version: ^9.5.1
+  version: ^9.8.0
```

Comment:
shouldn't the ^ be enough  ? Also maybe we should update here to dep (not necesarily to do it now, but maybe open an issue)

Reply:
I had tried to migrate this to dep a few weeks ago but I hit some issues so shelved it.  I will open an issue to track that.

Reply:
Oh wait I did get dep working here, it's in the v3 branch.

Reply:
@jhendrixMSFT  👍   . Please mention also any problem you encountered if you remember in the issue.